layouts/page/superpowers.html

```diff
@@ -0,0 +1,276 @@
+{{ define "hero" }}
+    <header class="header-hero shadow-2xl relative overflow-hidden">
+        <div class="absolute hero-clouds w-full"><img src="/images/hero-clouds.svg" alt=""></div>
+        <div class="container mx-auto">
+            <div class="flex items-start">
+                <div>
+                    <img class="w-64" src="/images/mascot/pulumipus.svg" alt="">
+                </div>
+                <div class="md:ml-8 md:mt-8 md:w-3/4">
+                    <h1 class="text-4xl">Modern Infrastructure with <span class="text-blue-300 font-semibold">Superpowers</span>.</h1>
+                    <p class="text-xl pr-48 text-purple-100 leading-relaxed">
+                        We're unleashing new capabilities for Delivery, Provisioning,
+                        Architecture, Testing and Policy-as-code &mdash; on any cloud, using your
+                        favorite languages. Join us as we walk through our favorite new features.
```

Comment:
What if a user wants to get started right away? (The ideal case!) "Join us" implies they can't do that yet?

Reply:
I'll adjust the wording here to be more immediately inviting to what's on the page now.

Reply:
> > Love it -- you mean on the home page, right? Or on the superpowers page?
> 
> I meant on the homepage, but possibly both would be nice 😄

Agree that some sparkle on /home would be great too.